test/integration/policy_imagemanifest_test.go

```diff
@@ -0,0 +1,189 @@
+		Eventually(
+			func() string {
+				output, _ := common.OcManaged("get", "subscriptions.operators.coreos.com", "-n",
+					operatorNS, subName, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("subscription.operators.coreos.com/container-security-operator"))
+	})
+
+	It("The operator should exist", func() {
+		By("Checking the operator in the " + operatorNS + " namespace")
+		Eventually(
+			func() string {
+				output, _ := common.OcManaged("get", "operators", subName+"."+operatorNS, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("operator.operators.coreos.com/container-security-operator.openshift-operators"))
+	})
+
+	It("stable/"+policyIMVName+" should detect imageManifestVulns", func() {
+		By("Checking if the status of the subscription config policy is NonCompliant")
+		Eventually(
+			utils.GetWithTimeout(
+				clientManagedDynamic,
+				common.GvrConfigurationPolicy,
+				"policy-imagemanifestvuln-sub",
+				clusterNamespace,
+				true,
+				defaultTimeoutSeconds,
+			).Object["status"].(map[string]interface{})["compliant"],
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.Compliant))
+
+		By("Checking if the status of the CSV config policy is NonCompliant")
+		Eventually(
+			utils.GetWithTimeout(
+				clientManagedDynamic,
+				common.GvrConfigurationPolicy,
+				"policy-imagemanifestvuln-status",
+				clusterNamespace,
+				true,
+				defaultTimeoutSeconds,
+			).Object["status"].(map[string]interface{})["compliant"],
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.Compliant))
+
+		By("Checking if the status of the IMV config policy is NonCompliant")
+		Eventually(
+			utils.GetWithTimeout(
+				clientManagedDynamic,
+				common.GvrConfigurationPolicy,
+				"policy-imagemanifestvuln-example-imv",
+				clusterNamespace,
+				true,
+				defaultTimeoutSeconds,
+			).Object["status"].(map[string]interface{})["compliant"],
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Cleans up", func() {
+		_, err := utils.KubectlWithOutput(
+			"delete", "-f", policyIMVURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		_, err = utils.KubectlWithOutput(
+			"delete", "subscription", subName, "-n", userNamespace, "--kubeconfig="+kubeconfigManaged,
+		)
+		Expect(err).To(BeNil())
+
+		csvName, err := utils.KubectlWithOutput(
+			"get", "clusterserviceversions",
+			"-o",
+			"jsonpath='{.items[?(@.spec.displayName==\"Quay Container Security\")].metadata.name}'",
```

Comment:
add `-n openshift-operators`. removing the one in this namespace should automatically remove the csv in all namespaces.